test/index.js

```diff
@@ -441,9 +441,8 @@ function fix() {
  * Argument Parsing
  */
 
-function parseArg() {
-  var argv = process.argv.slice(2),
```

Comment:
This looks like a bug to me. Did you intend to remove the `slice(2)` here?

Reply:
It seemed it was meant to get the argv from here. https://github.com/markedjs/marked/blob/d96d1d37cdd94e1e6fa100cea9a2b8ccb202b0ca/bin/marked#L81 because main(argv) didn't use the argv parameter. https://github.com/markedjs/marked/blob/d96d1d37cdd94e1e6fa100cea9a2b8ccb202b0ca/test/index.js#L553 It would just make the code nicer, wouldn't change functionality. I tested it and nothing changed.

Reply:
This code was added in 62467849142f92d85f2a7b7f1005fd5542fba4ae and it seems like `slice(2)` was intended because the first two arguments are typically `["node", "test.js"]`

/cc @UziTech

Reply:
It doesn't seem like it matters because "marked sdfg --test" works

Reply:
To prevent a breaking change we should slice the first two args off of `argv`.

```js
function parseArg(argv) {
  argv = argv.slice(2);
```